helm/main/values-101ed4-dev.yaml

```diff
@@ -22,6 +22,7 @@ app:
     "AWS_ROLES_BASE_URL": "https://dev.loginproxy.gov.bc.ca/auth"
     "AWS_ROLES_IDENTITY_PROVIDER": "azureidir"
     "AWS_ROLES_REALM_NAME": "public-cloud"
+    "M365_URL": "https://graph.microsoft.com"
```

Comment:
since `M365_URL` is the same for all environments, let's define it in `helm/main/values.yaml` only.